R/hiper_glm.R

```diff
@@ -1,31 +1,104 @@
+
+####################### Helper functions ######################
```

Comment:
-2

Use different files and roxygen function documentation to separate things out and document whats going on